### PR TITLE
Added ability to play a land card.

### PR DESCRIPTION
**Pull request by courageousillumination**
In an effort to get smaller PRs this does one thing: you can now play land. There's no rules checking so you can play lands at instant speed and as many as you want per turn. That's coming, but I didn't want to roll too much into this PR.

### CODE REVIEW

File: server/deckr/game/card.py [hunk 1]
```diff
@@ -55,6 +55,13 @@ def reset(self):
 
         pass
 
+    def is_land(self):
+        """
+        Check if the card is a land of any type.
+        """
+
+        return 'Land' in self.types
```
Comment (alexthebake) [inline server/deckr/game/card.py]: Quick question: where do we get our card data from? I was thinking we would use [DeckBrew](https://api.deckbrew.com/mtg/cards?name=forest), or something similar.

**Comment by alexthebake** [comment]
Looks good to me, we just need to fix the merge conflicts, and we can go ahead and merge this.

File: server/deckr/game/card.py (hunk 1 above)
Comment (courageousillumination) [inline server/deckr/game/card.py]: Right now I'm just hand coding it. I was planning to just use http://mtgjson.com/ but we could do something else.

Comment (alexthebake) [inline server/deckr/game/card.py]: Well, the cool thing about DeckBrew is that it actually pulls from mtgjson, and simply provides a convenient API

**Pull request merged**